spec/features/admin/admin_creates_new_admin_account_spec.rb

```diff
@@ -2,20 +2,18 @@
 
 feature "admin creates new admin account" do
   scenario "successfully" do
-    skip
     user = create(:user, admin: true)
     log_in_as user
 
-    visit new_user_path
-
+    visit new_admin_user_path
     fill_in "user_name", with: "Mary"
     fill_in "user_email", with: "Mary@example.com"
-    check "user_admin"
+    choose "Yes"
     fill_in "user_password", with: "secret"
-    click_on I18n.t("helpers.submit.user.create")
+    click_on I18n.t("admin.users.new.submit")
 
     expect(user.admin).to eq true
```

Comment:
We usually just check things that are on the page, so I would check that the page displays something like: `Admin: true` (or however it is displayed on the page).

If you decide to leave this assertion instead, I would change it to read:
`expect(user).to be_admin`

Reply:
Done in sha: 281932b